features/specs/meerderjarig-verklaarde-minderjarige.feature

```diff
@@ -0,0 +1,207 @@
+
+    Voorbeeld: Het kind is erkend voordat de moeder meerderjarig is verklaard
+      Gegeven de 17 jaar geleden geboren persoon 'Gerda'
+      En de 19 jaar geleden geboren persoon 'Aart'
+      En de 3 maanden geleden geboren persoon 'Bert' met één ouder 'Gerda'
+      En 2 maanden geleden is 'Bert' erkend door 'Aart'
+      En 1 maand geleden is 'Gerda' meerderjarig verklaard
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' eenhoofdig ouderlijk gezag met ouder 'Gerda'
+
+    Voorbeeld: Het kind is erkend voordat de moeder meerderjarig is geworden
+      Gegeven de 19 jaar geleden geboren persoon 'Gerda'
+      En de 21 jaar geleden geboren persoon 'Aart'
+      En de 2 jaar geleden geboren persoon 'Bert' met één ouder 'Gerda'
+      En 2 jaar geleden is 'Bert' erkend door 'Aart'
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' eenhoofdig ouderlijk gezag met ouder 'Gerda'
+
+    Voorbeeld: Het kind is erkend op een moment dat het gezag was toegewezen aan een voogd
+      Gegeven de 19 jaar geleden geboren persoon 'Gerda'
+      En de 21 jaar geleden geboren persoon 'Aart'
+      En de 1 jaar geleden geboren persoon 'Bert' met één ouder 'Gerda'
+      En 11 maanden geleden is het gezag toegewezen aan een voogd
+      En 8 maanden geleden is 'Bert' erkend door 'Aart'
+      En 2 maanden geleden is in een gerechtelijke uitspraak de voogdij beëindigd
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' eenhoofdig ouderlijk gezag met ouder 'Gerda'
+
+    Voorbeeld: Het kind is erkend op een moment dat de moeder onder curatele stond
+      Gegeven de 19 jaar geleden geboren persoon 'Gerda'
+      En de 21 jaar geleden geboren persoon 'Aart'
+      En de 3 jaar geleden geboren persoon 'Bert' met één ouder 'Gerda'
+      En 2 jaar geleden is 'Gerda' meerderjarig verklaard
+      En 1 jaar geleden is 'Gerda' onder curatele gesteld
+      En 8 maanden geleden is 'Bert' erkend door 'Aart'
+      En 1 maand geleden is de curatele voor 'Gerda' beëindigd
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' eenhoofdig ouderlijk gezag met ouder 'Gerda'
+
+    Voorbeeld: Het kind is erkend nadat de moeder is overleden
+      Gegeven de 19 jaar geleden geboren persoon 'Gerda'
+      En de 21 jaar geleden geboren persoon 'Aart'
+      En de 2 jaar geleden geboren persoon 'Bert' met één ouder 'Gerda'
+      En 2 jaar geleden is 'Gerda' meerderjarig verklaard
+      En 3 maanden geleden is 'Gerda' overleden
+      En 1 maand geleden is 'Bert' erkend door 'Aart'
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' tijdelijk geen gezag met de toelichting 'Tijdelijk geen gezag omdat de ouder overleden is.'
+
+  Regel: Als een ouder erkend heeft op een moment dat de erkenner niet bevoegd was tot gezag, dan kan er geen gezamenlijk gezag met de erkenner ontstaan
```

Comment:
dit geldt ook bij trouwen: als een van de ouders onder curatele staat op het moment van trouwen en daarna wordt curatele beëindigd: dan ontstaat door beëindigen van curatele geen gezamenlijk gezag --> nieuwe regel hiervoor maken